### PR TITLE
contrib/tcpdump: new package (4.99.4)

### PR DESCRIPTION
**Pull request by rbgarga**

### CODE REVIEW

**Comment by nekopsykose** [comment]
new port -> new package

File: contrib/tcpdump/template.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "--enable-smb",
+    "--with-crypto",
+]
+configure_gen = []
+make_cmd = "gmake"
+make_dir = "."
+hostmakedepends = ["gmake", "linux-headers", "pkgconf"]
+makedepends = ["libpcap-devel", "openssl-devel"]
+checkdepends = ["perl"]
+pkgdesc = "Ubiquitous network traffic analysis tool"
```
Comment (nekopsykose) [inline contrib/tcpdump/template.py:15]:
```suggestion
pkgdesc = "Command-line network traffic analysis tool"
```

File: contrib/tcpdump/template.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+pkgname = "tcpdump"
+pkgver = "4.99.4"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    "--enable-smb",
+    "--with-crypto",
+]
+configure_gen = []
```
Comment (nekopsykose) [inline contrib/tcpdump/template.py:9]: you can also use build_style =cmake here instead and then you don't need to worry about this or the make_dir/cmd stuff (just make_check_target = "check")

**Comment by rbgarga** [comment]
> new port -> new package

this time I used `new` in lowercase at least :P

**Comment by nekopsykose** [comment]
thanks a lot. was wondering where tcpdump was the other day but didn't have time to get to it :)

**Pull request merged**